### PR TITLE
Input parser

### PR DESCRIPTION
**Pull request by iefserge**
New method that creates embeds from string input, so it supports both embed codes and plain urls.
R = @kesla 
T = minor

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,11 @@
   "description": "Parse & render embeds",
   "main": "dist/index.js",
   "scripts": {
+    "lint": "semistandard | snazzy",
     "test": "nyc ava && semistandard | snazzy",
-    "prepublish": "mkdir -p dist && babel lib --out-dir dist"
+    "build": "babel lib --out-dir dist",
+    "watch": "babel lib --out-dir dist --watch",
+    "prepublish": "mkdir -p dist && npm run build"
```
Comment (kesla) [inline package.json:11]: ty

**Comment by kesla** [comment]
@iefserge but this means that if you, for example, adds a tweet with the embed code we're not going to parse it as fully as we could've done if we had used the `parse` method. There are things that you can't parse from only a url... Perhaps these are separated use-cases - but it would be cool if we could reuse more code between this implementation and the `.parse` method. I'm fine with it being this way right now, we can figure out this later on... Just something to think about.

I merged #2 - so there's now a readme if you rebase on latest master - perhaps add documentation there?

**Comment by iefserge** [comment]
@kesla thanks, yeah good idea, for now it's purpose is to determine whether embed is allowed or not, actual parse result is less important. I feel like it would be very cool to be able to just paste a twitter link and get the embed automatically, without searching for the embed code. Anyway, for now I'm going to keep using existing CMS embeds logic and use this for whitelisting only.

**Comment by kesla** [comment]
@iefserge thanks for explaining!

**Pull request merged**